### ISSUE TITLE
saga.attributes

### ISSUE THREAD

**Issue by andre-merzky** [radical-cybertools/radical.utils#4]
From Matteo: _Shall we migrate saga.attributes into radical.utils?_

From Andre: Yeah, I pondered about that, but at the end hesitated. Point is that the exceptions raised by the attrib interface are relatively SAGA specific -- if we'd move it to util, we would need to change to plain python exceptions -- but then would need a wrapper in saga to translate back to SAGA exceptions. But well, as it is now its also not perfect -- we eventually would need to translate the SAGA exceptions into plain python exceptions in Troy :P Although this can probably be solved more generically with proper exception inheritance on the SAGA end.

Long story short: this probably makes sense, and will happen someday -- I am still torn on the exception rendering. I'll leave this ticket open meanwhile...
